[FR/EN] Add `Help Centre/Client` and clean up the original article (#5858)
ppy/osu-wiki: wiki/Help_Centre/Client/en.md

4. Close osu!.
5. Find the `osu!.db` file located in the folder you just opened.
6. Right-click on it, then click `Rename`.
7. Rename it as you wish. It doesn't matter, as long as it is not called "osu!". Then press `Enter`.
8. Relaunch osu!.

> *Note: since version [Stable 20210519.3](https://osu.ppy.sh/home/changelog/stable40/20210519.3) dated 2021-05-19, this problem seems to become less frequent.*

Reviewer: Has it actually been confirmed to still be happening ever? We put a lot of effort into making sure it was fixed once-and-for-all, so may be worth removing this completely if that's the case?

Author: In any case for me this problem has not appeared since the last version, because I was downloading a lot of beatmaps at the same time, and I often had this status problem, and since this update, it has never come back.

After that I hesitated to delete this section completely, because if the problem comes back, there is no solution in the Help center, which would be annoying. 
But what might be possible would be to keep it in the article but put it between `<!-- -->` like what has already been done in `Help_centre/Website`, which seems to me to be a good compromise; not to totally delete it, and not to keep it either.

Reviewer: Let's leave it for the time being, I guess. How about changing this note to the following:

```suggestion
*Note: a fix for this was added in [Stable 20210519.3](https://osu.ppy.sh/home/changelog/stable40/20210519.3) dated 2021-05-19. If you happen to still encounter this issue, please let us know at https://github.com/ppy/osu-stable-issues/issues.*
```

Author: I agree.

> *Note: a fix for this was added in [Stable 20210519.3](https://osu.ppy.sh/home/changelog/stable40/20210519.3) dated 2021-05-19. If you happen to still encounter this issue, please let us know [here](https://github.com/ppy/osu-stable-issues/issues).*

Reviewer 2: ```suggestion
*Note: a fix for this was added in [Stable 20210519.3](https://osu.ppy.sh/home/changelog/stable40/20210519.3) dated 2021-05-19. If you happen to still encounter this issue, please [let us know](https://github.com/ppy/osu-stable-issues/issues).*
```

#### Allowing a program through AVG Firewall

> Please consult AVG's [official knowledgebase entry on the topic](https://support.avg.com/search?l=en&retUrl=SupportArticleView&searchText=Allow+application+in+AVG+Firewall&supportType=home) for detailed instructions on how to do this.

Reviewer 2: I'd rather have it link directly to the article, https://support.avg.com/SupportArticleView?l=en&urlName=AVG-Antivirus-Enhanced-Firewall-Application-Rules&supportType=home

> Please consult AVG's [official knowledgebase entry on the topic](https://support.avg.com/search?l=en&retUrl=SupportArticleView&searchText=Allow+application+in+AVG+Firewall&supportType=home) for detailed instructions on how to do this.

Author: ```suggestion
Please consult AVG's [official knowledgebase entry on the topic](https://support.avg.com/SupportArticleView?l=en&urlName=AVG-Antivirus-Enhanced-Firewall-Application-Rules&supportType=home) for detailed instructions on how to do this.
```

You can perform a trace route by opening Command Prompt and running the command `tracert`.

> To open the Command Prompt, press and hold the `Windows` key and press `R`, and then type `cmd` and click `OK`. [For more information, see this Windows knowledgebase article.](https://support.microsoft.com/en-us/topic/how-to-use-tracert-to-troubleshoot-tcp-ip-problems-in-windows-e643d72b-2f4f-cdd6-09a0-fd2989c7ca8e).

Reviewer 2: ```suggestion
To open the Command Prompt, press and hold the `Windows` key and press `R`, and then type `cmd` and click `OK`. For more information, see the [Windows knowledgebase article](https://support.microsoft.com/en-us/topic/how-to-use-tracert-to-troubleshoot-tcp-ip-problems-in-windows-e643d72b-2f4f-cdd6-09a0-fd2989c7ca8e).
```

**The servers could be having issues.**

> Check the [osu! server status page](https://status.ppy.sh/ "status.ppy.sh") to see if there are any ongoing issues with the servers.

Reviewer 2: ```suggestion
Check the [osu! server status page](https://status.ppy.sh/ "osu! server status") to see if there are any ongoing issues with the servers.
```
